src/main/java/cam72cam/prospectus/ProspectorPickItem.java

```diff
@@ -43,7 +43,7 @@ public EnumActionResult onItemUse(EntityPlayer player, World world, BlockPos pos
 		Map<String, Integer> counts = new HashMap<String, Integer>();
 		
 		for (int x = -radius; x < radius; x++) {
-			for (int y = 0; y < radius; y++) {
+			for (int y = -radius; y < radius; y++) {
```

Comment:
This was deliberately set to 0, it only scans below the current level.  This makes it easier to find at what level an ore stops appearing.

Reply:
:+1: